### ISSUE TITLE
Username not showing properly on comments if username contains quotes (")

### ISSUE THREAD

**Issue by elhananjair** [nextcloud/server#26861]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Open a side bar for any files
2. Try to mentioning someone and select that person from the list 
3. Press Enter key to see what will be seen as a comment 

### Expected behaviour
The mentioned persons photo should be appear as small Icon and thats person's name too 

### Actual behaviour
The name is wrapped with [&quot;] characters and the profile pic wont show up.
### Server configuration

**Operating system:**
Linux Fedora Server 33
**Web server:**
Apache
**Database:**
PostgreSQL
**PHP version:**
7.4.16
**Nextcloud version:** (see Nextcloud admin page)
21.0.1
**Updated from an older Nextcloud/ownCloud or fresh install:**
No
**Where did you install Nextcloud from:**
From Official Website 
**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
No
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No
#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**
Microsoft Edge
**Operating system:**
Windows 10
#### Web server error log
<details>
<summary>Web server error log</summary>

```
[Mon May 03 07:25:55.980604 2021] [core:notice] [pid 795:tid 795] SELinux policy enabled; httpd running as context system_u:system_r:httpd_t:s0
[Mon May 03 07:25:56.010797 2021] [suexec:notice] [pid 795:tid 795] AH01232: suEXEC mechanism enabled (wrapper: /usr/sbin/suexec)
[Mon May 03 07:25:56.025875 2021] [ssl:error] [pid 795:tid 795] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 07:25:56.025894 2021] [ssl:error] [pid 795:tid 795] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using localhost.localdomain. Set the 'ServerName' directive globally to suppress this message
[Mon May 03 07:25:56.048418 2021] [ssl:error] [pid 795:tid 795] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 07:25:56.048434 2021] [ssl:error] [pid 795:tid 795] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
[Mon May 03 07:25:56.048521 2021] [lbmethod_heartbeat:notice] [pid 795:tid 795] AH02282: No slotmem from mod_heartmonitor
[Mon May 03 07:25:56.054558 2021] [mpm_event:notice] [pid 795:tid 795] AH00489: Apache/2.4.46 (Fedora) OpenSSL/1.1.1k configured -- resuming normal operations
[Mon May 03 07:25:56.054604 2021] [core:notice] [pid 795:tid 795] AH00094: Command line: '/usr/sbin/httpd -D FOREGROUND'
[Mon May 03 08:07:31.057973 2021] [access_compat:error] [pid 905:tid 1034] [client 192.168.15.19:51445] AH01797: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Mon May 03 08:11:42.603146 2021] [access_compat:error] [pid 903:tid 956] [client 192.168.15.19:51786] AH01797: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Mon May 03 09:08:51.783888 2021] [mpm_event:notice] [pid 795:tid 795] AH00492: caught SIGWINCH, shutting down gracefully
[Mon May 03 09:09:43.444890 2021] [core:notice] [pid 761:tid 761] SELinux policy enabled; httpd running as context system_u:system_r:httpd_t:s0
[Mon May 03 09:09:43.484244 2021] [suexec:notice] [pid 761:tid 761] AH01232: suEXEC mechanism enabled (wrapper: /usr/sbin/suexec)
[Mon May 03 09:09:43.499577 2021] [ssl:error] [pid 761:tid 761] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:09:43.499598 2021] [ssl:error] [pid 761:tid 761] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using localhost.localdomain. Set the 'ServerName' directive globally to suppress this message
[Mon May 03 09:09:43.523390 2021] [ssl:error] [pid 761:tid 761] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:09:43.523411 2021] [ssl:error] [pid 761:tid 761] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
[Mon May 03 09:09:43.523525 2021] [lbmethod_heartbeat:notice] [pid 761:tid 761] AH02282: No slotmem from mod_heartmonitor
[Mon May 03 09:09:43.528941 2021] [mpm_event:notice] [pid 761:tid 761] AH00489: Apache/2.4.46 (Fedora) OpenSSL/1.1.1k configured -- resuming normal operations
[Mon May 03 09:09:43.528983 2021] [core:notice] [pid 761:tid 761] AH00094: Command line: '/usr/sbin/httpd -D FOREGROUND'
[Mon May 03 09:09:43.594340 2021] [proxy:error] [pid 872:tid 934] (2)No such file or directory: AH02454: FCGI: attempt to connect to Unix domain socket /run/php-fpm/www.sock (*) failed
[Mon May 03 09:09:43.594368 2021] [proxy_fcgi:error] [pid 872:tid 934] [client 192.168.15.19:52160] AH01079: failed to make connection to backend: httpd-UDS
[Mon May 03 09:09:43.946335 2021] [proxy:error] [pid 875:tid 1021] (2)No such file or directory: AH02454: FCGI: attempt to connect to Unix domain socket /run/php-fpm/www.sock (*) failed
[Mon May 03 09:09:43.946391 2021] [proxy_fcgi:error] [pid 875:tid 1021] [client 192.168.20.32:58420] AH01079: failed to make connection to backend: httpd-UDS
[Mon May 03 09:32:15.973389 2021] [access_compat:error] [pid 873:tid 1078] [client 192.168.5.86:56413] AH01797: client denied by server configuration: /var/www/html/nextcloud/config
[Mon May 03 09:38:14.158310 2021] [access_compat:error] [pid 873:tid 1003] [client 192.168.15.19:52333] AH01797: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Mon May 03 09:45:09.283682 2021] [mpm_event:notice] [pid 761:tid 761] AH00492: caught SIGWINCH, shutting down gracefully
[Mon May 03 09:45:59.172830 2021] [core:notice] [pid 759:tid 759] SELinux policy enabled; httpd running as context system_u:system_r:httpd_t:s0
[Mon May 03 09:45:59.211908 2021] [suexec:notice] [pid 759:tid 759] AH01232: suEXEC mechanism enabled (wrapper: /usr/sbin/suexec)
[Mon May 03 09:45:59.228743 2021] [ssl:error] [pid 759:tid 759] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:45:59.228762 2021] [ssl:error] [pid 759:tid 759] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using localhost.localdomain. Set the 'ServerName' directive globally to suppress this message
[Mon May 03 09:45:59.252138 2021] [ssl:error] [pid 759:tid 759] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:45:59.252172 2021] [ssl:error] [pid 759:tid 759] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
[Mon May 03 09:45:59.252273 2021] [lbmethod_heartbeat:notice] [pid 759:tid 759] AH02282: No slotmem from mod_heartmonitor
[Mon May 03 09:45:59.257724 2021] [mpm_event:notice] [pid 759:tid 759] AH00489: Apache/2.4.46 (Fedora) OpenSSL/1.1.1k configured -- resuming normal operations
[Mon May 03 09:45:59.257771 2021] [core:notice] [pid 759:tid 759] AH00094: Command line: '/usr/sbin/httpd -D FOREGROUND'
[Mon May 03 09:45:59.313899 2021] [proxy:error] [pid 829:tid 843] (2)No such file or directory: AH02454: FCGI: attempt to connect to Unix domain socket /run/php-fpm/www.sock (*) failed
[Mon May 03 09:45:59.313939 2021] [proxy_fcgi:error] [pid 829:tid 843] [client 192.168.15.19:52421] AH01079: failed to make connection to backend: httpd-UDS
[Mon May 03 09:46:18.564867 2021] [access_compat:error] [pid 831:tid 1032] [client 192.168.5.86:57017] AH01797: client denied by server configuration: /var/www/html/nextcloud/config
[Mon May 03 09:47:58.699820 2021] [mpm_event:notice] [pid 759:tid 759] AH00492: caught SIGWINCH, shutting down gracefully
[Mon May 03 09:48:51.262619 2021] [core:notice] [pid 758:tid 758] SELinux policy enabled; httpd running as context system_u:system_r:httpd_t:s0
[Mon May 03 09:48:51.316961 2021] [suexec:notice] [pid 758:tid 758] AH01232: suEXEC mechanism enabled (wrapper: /usr/sbin/suexec)
[Mon May 03 09:48:51.335294 2021] [ssl:error] [pid 758:tid 758] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:48:51.335315 2021] [ssl:error] [pid 758:tid 758] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using localhost.localdomain. Set the 'ServerName' directive globally to suppress this message
[Mon May 03 09:48:51.358722 2021] [ssl:error] [pid 758:tid 758] AH02217: ssl_stapling_init_cert: can't retrieve issuer certificate! [subject: OU=root@localhost.localdomain,O=mkcert development certificate / issuer: CN=mkcert root@localhost.localdomain,OU=root@localhost.localdomain,O=mkcert development CA / serial: D986BC4FA64F68273CEECD0287D60358 / notbefore: Jan 27 08:37:30 2021 GMT / notafter: Jan 27 08:37:30 2031 GMT]
[Mon May 03 09:48:51.358744 2021] [ssl:error] [pid 758:tid 758] AH02604: Unable to configure certificate nextacso.com:443:0 for stapling
[Mon May 03 09:48:51.358842 2021] [lbmethod_heartbeat:notice] [pid 758:tid 758] AH02282: No slotmem from mod_heartmonitor
[Mon May 03 09:48:51.363858 2021] [mpm_event:notice] [pid 758:tid 758] AH00489: Apache/2.4.46 (Fedora) OpenSSL/1.1.1k configured -- resuming normal operations
[Mon May 03 09:48:51.363900 2021] [core:notice] [pid 758:tid 758] AH00094: Command line: '/usr/sbin/httpd -D FOREGROUND'
[Mon May 03 09:50:19.710336 2021] [access_compat:error] [pid 860:tid 1039] [client 192.168.15.19:52477] AH01797: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Mon May 03 09:50:34.588945 2021] [access_compat:error] [pid 860:tid 1040] [client 192.168.15.19:52477] AH01797: client denied by server configuration: /var/www/html/nextcloud/data/.ocdata
[Mon May 03 09:58:16.536885 2021] [access_compat:error] [pid 1677:tid 1733] [client 192.168.5.86:49721] AH01797: client denied by server configuration: /var/www/html/nextcloud/config
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"YI@lcJ--sqwTci6slAIkJQAAzgE","level":2,"time":"2021-05-03T07:25:36+00:00","remoteAddr":"192.168.5.37","user":"Chaltu Negera","app":"filesystem","method":"GET","url":"/ocs/v2.php/apps/notifications/api/v2/notifications","message":"Storage wrapper 'sharepermissions' was not registered via the 'OC_Filesystem - preSetup' hook which could cause potential problems.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36 Edg/90.0.818.51","version":"21.0.1.1"}
{"reqId":"YI@lcJ--sqwTci6slAIkJgAAzg8","level":2,"time":"2021-05-03T07:25:36+00:00","remoteAddr":"192.168.5.37","user":"Chaltu Negera","app":"filesystem","method":"GET","url":"/ocs/v2.php/apps/spreed/api/v2/room","message":"Storage wrapper 'sharepermissions' was not registered via the 'OC_Filesystem - preSetup' hook which could cause potential problems.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36 Edg/90.0.818.51","version":"21.0.1.1"}
{"reqId":"YI@lhiTvbzel1fYofPcGcAAARQk","level":2,"time":"2021-05-03T07:25:58+00:00","remoteAddr":"192.168.15.19","user":"Amanuel Kumera","app":"filesystem","method":"GET","url":"/ocs/v2.php/apps/notifications/api/v2/notifications","message":"Storage wrapper 'sharepermissions' was not registered via the 'OC_Filesystem - preSetup' hook which could cause potential problems.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.49","version":"21.0.1.1"}
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
index.js:46 No OC found
t.getLanguage @ index.js:46
jquery-migrate.min.js:2 JQMIGRATE: Migrate is installed, version 3.3.2
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
6globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
6globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
globals.js:60 Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20
pe @ globals.js:60
activity-sidebar.js?v=1f1b1e53-8:603 Uncaught TypeError: Cannot read property 'extend' of undefined
    at activity-sidebar.js?v=1f1b1e53-8:603
    at activity-sidebar.js?v=1f1b1e53-8:803
2globals.js:60 Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20
pe @ globals.js:60
versionstabview.js:18 Uncaught TypeError: Cannot read property 'extend' of undefined
    at Module.<anonymous> (versionstabview.js:18)
    at n (bootstrap:19)
    at bootstrap:83
    at files_versions.js?v=1f1b1e53-8:1
3globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
session-heartbeat.js:97 session heartbeat polling started
Viewer.vue:235 Please do NOT wait for the DOMContentLoaded before registering your viewer handler
handlers @ Viewer.vue:235
3globals.js:60 $ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
preview:1 Failed to load resource: the server responded with a status of 404 ()
RecommendedFile.vue:110 could not load recommendation preview Event
n.onerror @ RecommendedFile.vue:110
preview:1 Failed to load resource: the server responded with a status of 404 ()
RecommendedFile.vue:110 could not load recommendation preview Event
n.onerror @ RecommendedFile.vue:110
index.js:311 OC.requestToken changed m8RIiMu/vnniuvfqk2oSBTozfL+3HjO0cGWmkhLjhkA=:7q57varSiiHb9L2D/AF8PUBZLc//TwSGHSnE+nbIqSQ=
Viewer.js:122 Opening the viewer with a single string parameter is deprecated. Please use a destructuring object instead OCA.Viewer.open({ path: '/Test for Acso Document/Acso Documents/Readme.md' })
value @ Viewer.js:122
Viewer.vue:248 Opening viewer for file  /Test for Acso Document/Acso Documents/Readme.md
util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
2capabilities.js:32 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:32
SharingInput.vue:311 recommendations Array(5)
2util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
index.js:311 OC.requestToken changed UBc6gsS8ACkUMW1yBrlzVtudYoRD5Z/Ho0i+7egHFVc=:JX0Jt6XRNHEtfycbadIdbqH3M/QLtKj1zgTchYwsOjM=
6util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
Viewer.js:122 Opening the viewer with a single string parameter is deprecated. Please use a destructuring object instead OCA.Viewer.open({ path: '/CSO Documents/2901/Readme.md' })
value @ Viewer.js:122
Viewer.vue:248 Opening viewer for file  /CSO Documents/2901/Readme.md
util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
2capabilities.js:32 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:32
SharingInput.vue:311 recommendations Array(5)
preview:1 Failed to load resource: the server responded with a status of 404 ()
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
Viewer.js:122 Opening the viewer with a single string parameter is deprecated. Please use a destructuring object instead OCA.Viewer.open({ path: '/Test for Acso Document/Acso Documents/Readme.md' })
value @ Viewer.js:122
Viewer.vue:248 Opening viewer for file  /Test for Acso Document/Acso Documents/Readme.md
util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
2capabilities.js:32 OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities
getCapabilities @ capabilities.js:32
SharingInput.vue:311 recommendations Array(5)
util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
util.js:133 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment
relativeModifiedDate @ util.js:133
(anonymous) @ init.js:96
each @ jquery.js:354
each @ jquery.js:189
(anonymous) @ init.js:95
2globals.js:60 jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own.
pe @ globals.js:60
index.js:311 OC.requestToken changed lbMQPg8GQxU0L902+wdE5zQ8naGeJFAinmWAnxw3YmY=:4NkjC25rd00NYZdflGwq305WzNHWdWcQ8yni93gcTQI=
```
</details>

![markdown_parse](https://user-images.githubusercontent.com/69236638/116851357-e5c50480-abfa-11eb-9b6f-5ad1b2f1105c.PNG)
![markdown_parse2](https://user-images.githubusercontent.com/69236638/116851375-e9f12200-abfa-11eb-94d9-f79ffa5afe44.PNG)

**Comment by artonge** [comment]
Hi,

Are you experiencing the problem in the comment input or in the comment list once the comment is submitted ?

**Comment by elhananjair** [comment]
> Hi,
> 
> Are you experiencing the problem in the comment input or in the comment list once the comment is submitted ?

Thanks for replying @artonge , as I point out on the pic I attached, When I write somebody's name by prepending '@', some search results will appear, and after I clicked on the result I wanted, @"The name of the guy" seen like this. That one is before I submit the comment. And After I click submit, the name will be seen like this, @"&quot;the name of the guy&quot; my message". 
This problem is the same in all apps that have comment menu.

**Comment by elhananjair** [comment]
the quotes inserted as I select the username from the list, and the quote + &qoute...insterted after I submit,

**Comment by artonge** [comment]
I manage to reproduce by creating a user with a username containing quotes. I'll see what we can do.

**Comment by elhananjair** [comment]
Here I attached a video to describe it more



> I manage to reproduce by creating a user with a username containing quotes. I'll see what we can do.

but a user am using here doesn't contain quotes

**Comment by artonge** [comment]
Thanks for reporting @elhananjair !

**Comment by artonge** [comment]
> @artonge I am still having this problem. NC 21.0.5

As I already said: NC 21.0.4 uses nextcloud/vue@v3.2.0. This fix has only made it to nextcloud/vue@v4.0.0 which will be used in NC 23

**Comment by acsfer** [comment]
Reopening as @elhananjair reports this isn't fixed ad NC 23.
cc @artonge

**Comment by elhananjair** [comment]
I found out that the problem is fixed now, but I can see the difference after I refresh the page. Check out this video.

https://user-images.githubusercontent.com/69236638/145382581-1d03b6b3-eb1e-4496-bcfb-b5403f127077.mp4

**Comment by acsfer** [comment]
Thanks; let's keep it open as this should be addressed too. 👍

**Comment by danxuliu** [comment]
> I found out that the problem is fixed now (v23), but I can see the difference **only after I refresh the page**. Check out this video.
> 2021-12-09.13-45-16.mp4

This should be fixed by #35054
